tools/tertiary-analysis/garnett/.shed.yml

```diff
@@ -0,0 +1,20 @@
+categories:
+ - Transcriptomics
+ - RNA
+ - Statistics
+ - Sequence Analysis
+description: "De-composed garnett functionality tools, based on https://github.com/ebi-gene-expression-group/garnett-cli and r-garnett 0.2.8"
```

Comment:
Same comments, capitalisation of Garnet, and odd use of 'based on'- and same for below.